pkg/application/utils.go

```diff
@@ -46,3 +47,27 @@ func parseTemplate(manifest string, config *ValuesTemplateVars) string {
 
 	return manifestBuffer.String()
 }
+
+const VersionOverrideEnvPrefix = "E2E_OVERRIDE_VERSIONS"
```

Comment:
is it really a prefix?

Reply:
sounds like normal env name based on the code below, but that's just minor nitpick